draft-jhoyla-req-mtls-flag.md

```diff
@@ -59,15 +59,44 @@ messages in the wild. A client sending the `request_client_auth` flag in the
 `ClientHello` allows the server to request authentication during the initial
 handshake only when it receives a hint the client will handle the request
 without terminating the connection, and, if unable to authenticate, by sending
-an empty `Certificate` message, per {{?RFC8446, Section 4.4.2}}. This enables a
-number of use cases, for example allowing bots to authenticate themselves when
-mixed in with general traffic.
-
-This flag is intended to be used by clients sending automated traffic, not
-those that are directly operated by (human) users. Clients that set this flag
-should only do so when a client certificate is directly available to them, and
-must not prompt the user or take actions that would prompt the user for example
-trying to read certificates from off-board devices such as smart cards.
+an empty `Certificate` message, per {{?RFC8446, Section 4.4.2}}.
+
+This mechanism is primarily intended to allow web servers to cryptographically
+authenticate web crawlers and other trusted sources of automated traffic
+without prompting human users to present a certificate.
+
+The traffic generated by these sources represents a significant and ever
+increasing fraction of the overall traffic to a given web server. Many web
+servers handle this traffic differently than they handle traffic from web
+browsers operated by human users. On the one hand, human users are much more
+latency sensitive than web crawlers. On the other hand, it may be prudent to
+treat requests from an unknown client with more scrutiny than requests from a
+well-known web-crawler whose behaviour is relatively predictable.
+
+It is therefore common for web servers to attempt to identify trusted sources
+of traffic so that they can be distinguished between one another and,
+especially, from human users. This dynamic creates an incentive for attackers
```

Comment:
slightly changing the wording to make it clear we want to identify each crawler individually, and distinguish them from human users

```suggestion
of traffic so that they can be distinguished between one another, as well as from human users. This dynamic creates an incentive for attackers
```